web_ui/packages/smart-tools/src/ritm/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+export { buildRITMInstance, RITM, RITM_TEMPLATE_SIZE } from './ritm';
+export { RITMModels } from './models/models';
```

Comment:
I don't think we should expose this since it's not something we would want the consumer of the package to use directly: only `buildRITMInstance` and the interface types belong to our public interface.